components/SkilledImpact/elements/ApplicationContent.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, Switch } from 'react-native';
+import styles from '../../../constants/SkilledImpact/OrgSkilledImpactBody';
+import Sync from '../../../assets/jsicons/bottomnavigation/Sync';
+import ChevronBottom from '../../../assets/jsicons/miscIcons/ChevronBottom';
+import ChevronRight from '../../../assets/jsicons/miscIcons/ChevronRight';
+import ApplicationElement from './ApplicationElement';
+
+class ApplicationContent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mainExpanded: true,
+      noDecisionExpanded: true,
+      selectedExpanded: true,
+      notSelectedExpanded: true,
+    };
+  }
+
+  toggleMainExpand = () => {
+    this.setState({ mainExpanded: !this.state.mainExpanded });
+  };
+
+  toggleNoDecisionExpand = () => {
+    this.setState({ noDecisionExpanded: !this.state.noDecisionExpanded });
+  };
+
+  toggleSelectedExpand = () => {
+    this.setState({ selectedExpanded: !this.state.selectedExpanded });
+  };
+
+  toggleNotSelectedExpand = () => {
+    this.setState({ notSelectedExpanded: !this.state.notSelectedExpanded });
+  };
+
+  render() {
+    return (
+      <View style={styles.itemContainers}>
+        <TouchableOpacity
+          style={styles.itemTitleRow}
+          onPress={this.toggleMainExpand}
+        >
+          <Sync />
```

Comment:
This should be a page icon, take a look at the mockup in Monday. Right now it kinda looks like another section of applications, which is a little awkward.